app/chrony/README.md

```diff
@@ -2,21 +2,19 @@
 
 Basic chrony NTP client monitoring.
 
-* [Template App chrony active](Template_App_chrony_active.xml)
-
 This template is part of [RaBe's Zabbix template and helpers
 collection](https://github.com/radiorabe/rabe-zabbix).
 
-## Template
-
-### Items
-
-* Memory usage (rss) of "chronyd" processes (proc.mem[chronyd,chrony,,,rss])
-* Number of "chronyd" processes (proc.num[chronyd,chrony])
+## Template App chrony active
 
+### Items 
+* Memory usage ($5) of "$1" processes (`proc.mem[chronyd,chrony,,,rss]`)
+* Number of "$1" processes (`proc.num[chronyd,chrony]`)
 ### Triggers
 
-* High: No running chronyd processes on {HOST.NAME}
+* High: No running chronyd processes on {HOST.NAME} (`{Template App chrony active:proc.num[chronyd,chrony].max(#5)}<1`)
```

Comment:
You can add a description and it will output that on the line after (like with the items).

I had this in a table at one point but descriptions make the tables way too wide and lead to a side scrolling situation with is worse for the readability (at least on github).

It probably makes sense to change this into a table once we switch to something like mkdocs where we have more control over the page layout.